### PR TITLE
docs(linux): Update sample tasks.json for Linux

### PR DESCRIPTION
**Pull request by ermshiperete**
@keymanapp-test-bot skip

### CODE REVIEW

File: docs/settings/linux/tasks.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
       "type": "shell",
       "label": "ibus-keyman: build",
       "command": "./build.sh",
-      "args": [ "build", "--debug" ],
+      "args": [ "build", "--debug", "--coverage" ],
```
Comment (darcywong00) [inline docs/settings/linux/tasks.json:124]: Is `--coverage` only applicable to `test`?

Comment (jahorton) [reply docs/settings/linux/tasks.json:124]: Probably; code coverage is usually a measure of how much of the codebase is being tested, after all.

**Review by jahorton — APPROVED** [review]

File: docs/settings/linux/tasks.json (hunk 1 above)
Comment (ermshiperete) [reply docs/settings/linux/tasks.json:124]: Yes and no. You'll get code coverage only when you run tests, so yes, strictly speaking it's only applicable there. However, you also need to compile with specific flags, which is why I also added it to the `configure` and `build` actions.. I don't think it hurts to always compile with these flags during development when you build in VSCode - IMO less interrupting than trying to get the code coverage and then detecting that you'll have to clean and rebuild.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.182-alpha](https://keyman.com/downloads/releases/alpha/17.0.182)

**Pull request merged**